### PR TITLE
Allow FastAPI < 1.0.0 in requirements

### PR DESCRIPTION
**Pull request by Viicos**
Fixes https://github.com/zauberzeug/nicegui/issues/805.

I think 0.92 is a good lower bound, as it fixes a [security issue](https://github.com/tiangolo/fastapi/releases/tag/0.92.0) related to upload files.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -18,7 +18,7 @@ matplotlib = [
     { version = "^3.5.0", markers = "python_version ~= '3.7'"},
     { version = "^3.6.0", markers = "python_version ~= '3.11.0'"},
 ]
-fastapi = "^0.92"
+fastapi = ">=0.92,<1.0.0"
```
Comment (Viicos) [inline pyproject.toml:21]: I think `~0` could work here according to [docs](https://python-poetry.org/docs/dependency-specification/#tilde-requirements), but hurts readability.

Comment (falkoschindler) [reply pyproject.toml:21]: Yes, I agree. 👍

**Comment by falkoschindler** [comment]
Thanks for the quick PR, @Viicos!
It's already released with version 1.2.9. 🚀

**Comment by Viicos** [comment]
That was fast, thanks!

**Pull request merged**